experimental/nodejs/README.md

```diff
@@ -4,9 +4,11 @@
 
 This image contains a minimal Linux, Node.js-based runtime.
 
-Specifically, the image contains everything in the [base image](../../base/README.md), plus:
+Specifically, the image contains everything in the [base image](../../base/README.md), plus one of:
 
-- Node.js v10.16.3 and its dependencies.
+- Node.js v10 (`gcr.io/distroless/nodejs:10`) and its dependencies.
+- Node.js v12 (`gcr.io/distroless/nodejs:12`) and its dependencies.
+- Node.js v14 (`gcr.io/distroless/nodejs:14`) and its dependencies.
 
```

Comment:
It would be useful to have some text to describe the `latest` tag.  Does it make sense to deprecate `latest` and encourage users to explicitly track the specific LTS release they're interested in?  And remove `latest` when NodeJS v10 reaches EOL in 2021-04-30?

Reply:
Good point! I'm all for deprecating the latest tag since it is confusing.

I added a note explaining the `latest` tag and encouraging users to specify a LTS version. I also added another sentence at the top explaining that the supported images follow the Node.js LTS releases with a link.